examples/pylab_examples/stock_demo.py

```diff
@@ -1,19 +1,51 @@
 import matplotlib.pyplot as plt
 import numpy as np
 
+import matplotlib.cbook as cbook
 from matplotlib.ticker import MultipleLocator
-from data_helper import get_two_stock_data
+
+"""
+Stock Demo Plots
+
+The following example displays Matplotlibs capabilities of creating
+graphs that can be used for stocks. The example specfically uses
+Apple and Intel stock data and graphs the normalized prices on the
+same plot. 
+
+"""
+
+def get_two_stock_data():
+    """
+    load stock time and price data for two stocks The return values
+    (d1,p1,d2,p2) are the trade time (in days) and prices for stocks 1
+    and 2 (intc and aapl)
+    """
+    ticker1, ticker2 = 'INTC', 'AAPL'
+
+    file1 = cbook.get_sample_data('INTC.dat.gz')
+    file2 = cbook.get_sample_data('AAPL.dat.gz')
+    M1 = fromstring(file1.read(), '<d')
```

Comment:
These calls and following lines are missing the `np.` prefix (because the original `data_helper.py` imported them poorly.)

I think it can use `np.fromfile` directly instead of `fromstring` and a `read`.